src/runtime-rs/Makefile

```diff
@@ -168,9 +169,18 @@ DEFFILEMEMBACKEND := ""
 DEFVALIDFILEMEMBACKENDS := [\"$(DEFFILEMEMBACKEND)\"]
 DEFMSIZE9P := 8192
 DEFVFIOMODE := guest-kernel
+##VAR DEFSANDBOXCGROUPONLY=<bool> Default cgroup model
+DEFSANDBOXCGROUPONLY ?= false
+DEFSANDBOXCGROUPONLY_DB ?= true
+DEFSANDBOXCGROUPONLY_FC ?= true
+DEFSTATICRESOURCEMGMT ?= false
+DEFSTATICRESOURCEMGMT_DB ?= false
+DEFSTATICRESOURCEMGMT_FC ?= true
```

Comment:
`DEFSANDBOXCGROUPONLY` and `DEFSTATICRESOURCEMGMT` were [removed recently](https://github.com/kata-containers/kata-containers/pull/10052/commits/d11657a581243cef536b4c6cce6710bd22339bc9) - why are we adding them back?  @ananos can you please take a look?

Also, why is this PR touching dragonball and firecracker build?